pro/src/ui-kit/RadioButtonWithImage/RadioButtonWithImage.module.scss

```diff
@@ -41,10 +41,10 @@
     outline: 1px solid colors.$tertiary;
     font-weight: 700;
     background-color: rgba(colors.$tertiary, 0.05);
-
-    &:focus-within {
+    &:has(:focus-visible) {
```

Comment:
J'ai du mal à comprendre pourquoi tu as besoin du `has` ?

Reply:
Hmmm j'ai l'impression que le focus se fait sur un élément plus bas que  l'élement portant les classes `.button` `.is-selected` donc que le :focus-visible seul ne fonctionne pas

Reply:
Ah oui en effet, ok